adx

```diff
@@ -187,6 +187,11 @@ actions['test'] = subparsers.add_parser(
 actions['test'].set_defaults(func=util.run_tests)
 actions['test'].add_argument('extension', metavar='name', type=str,
                              help='The extension name e.g."validation", "ytp-request"')
+actions['test'].add_argument(
+    '--tests', '-t', type=str,
+    help='The test(s) to run e.g. test_actions.py::TestFormatGuess',
+    default=""
+)
```

Comment:
That's redundant as you can already pass pytest `-k` args specifying keywords to filter out test cases:
https://docs.pytest.org/en/6.2.x/usage.html#specifying-tests-selecting-tests
e.g.
```
# all test from a module
adx test validation -k test_custom_checks
# complex matching, double quotes needed so " are passed to docker-compose command
adx test validation -k '"test_custom_checks and test_check_custom_constraint_for_missing_data"'
```